src/main/resources/application.yaml

```diff
@@ -920,6 +920,7 @@ ftpaSubmitted:
 
 customerServices:
   emailAddress: ${IA_CUSTOMER_SERVICES_EMAIL:contactia@justice.gov.uk}
+  ftpaEmailAddress: ${IA_FTPA_CUSTOMER_SERVICES_EMAIL:CTSC-IAC-Teamleaders@justice.gov.uk}
```

Comment:
please do not put real email addresses in application code

Reply:
Replaced real email with IA-FTPA-Customer-Services@example.gov.uk